docs/content/policy-reference.md

```diff
@@ -844,7 +844,7 @@ The `request` object parameter may contain the following fields:
 | `headers` | no | `object` | HTTP headers to include in the request (e.g,. `{"X-Opa": "rules"}`). |
 | `enable_redirect` | no | `boolean` | Follow HTTP redirects. Default: `false`. |
 | `force_json_decode` | no | `boolean` | Decode the HTTP response message body as JSON even if the `Content-Type` header is missing. Default: `false`. |
-| `tls_use_system_certs` | no | `boolean` | Use the system certificate pool. Default: `false`. |
```

Comment:
Thanks for updating the docs! There's a note below around line 869 about this param; it would be good to update it to something like:

```
> By default `http.send` uses the system certificate pool to verify TLS server certificates. If `tls_ca_cert`, `tls_ca_cert_file`, or `tls_ca_cert_env_variable` are specified the system certificate pool is disabled unless explicitly enabled by passing `true` for the `tls_use_system_certs` key.
```